src/services/CreateAnalysisService.tsx

```diff
@@ -21,22 +21,28 @@ class CreateAnalysisService {
 
   static formatDate(dateStr: string): string {
     const date = new Date(dateStr);
-    return `${date.getFullYear()} ${date.getMonth() + 1} ${date.getDate()}`
+    let month: string = '';
+    let day: string = '';
+
+    if ((date.getMonth() + 1) < 10) {
+      month = '0' + (date.getMonth() + 1);
+    } else {
+      month = `${date.getMonth() + 1}`;
+    }
+
+    if ((date.getDate() + 1) < 10) {
+      day = '0' + (date.getDate() + 1);
+    } else {
+      day = `${date.getDate() + 1}`;
+    }
+
+    return `${date.getFullYear()} ${month} ${day}`
```

Comment:
I was wondering whether we had to follow the `<year> <month> <date>` format and whether we can get the string directly from the `Date` object? Have you looked into `Date.prototype.toLocaleDateString()` or other `Date` methods?

Reply:
Disregard `toLocaleDateString()`. We should leverage `moment` since it's already in the project.

Reply:
`moment().format("YYYY MM DD");`